### PR TITLE
Add frontend for system log

### PR DESCRIPTION
**Pull request by postlund**
This is the frontend code for the system log I'm implementing. See pull request in the Home Assistant repo for more details.

### CODE REVIEW

**Comment by balloob** [comment]
Future idea: add a report button to push stacktraces to us.

**Comment by balloob** [comment]
I don't think that this should be a panel but instead be integrated into the info dev tool.

**Comment by postlund** [comment]
Do you have any preferences on how you want it to look? Or should I just keep current layout and put it below the "line"?

**Comment by balloob** [comment]
I like that idea 👍

**Comment by postlund** [comment]
So, I've updated the frontend now and it looks something like this:

screenshot

I decided to change so that the full log is not automatically loaded and so that the user has to press refresh manually. It's quite sluggish if the log is large and should hopehully be less needed now with the new system log.

**Comment by postlund** [comment]
I have cleaned up a bit but an "issue" I have is that the `paper-dialog` is not scrollable (on desktop, works fine on my phone), so long output might not be possible to see. Been playing around with `paper-dialog-scrollable` but cannot get it to work. Any tips on what to do?

**Comment by balloob** [comment]
See how we use it here: https://github.com/home-assistant/home-assistant-polymer/blob/master/src/dialogs/more-info-dialog.html#L95

File: panels/dev-info/ha-panel-dev-info.html [hunk 1]
```diff
@@ -41,7 +46,6 @@
 
       .error-log-intro {
         margin-top: 16px;
-        border-top: 1px solid var(--light-primary-color);
```
Comment (balloob) [inline panels/dev-info/ha-panel-dev-info.html]: If you're removing the border, remove the padding top and set margin to 24px.

File: panels/dev-info/ha-panel-dev-info.html [hunk 2]
```diff
@@ -54,6 +58,56 @@
...
+      .row {
+        display: flex;
+        justify-content: space-between;
+      }
+
+      paper-dialog {
+        border-radius: 2px;
+      }
+
+      @media all and (max-width: 450px) {
```
Comment (balloob) [inline panels/dev-info/ha-panel-dev-info.html]: When the max-width is 450px, change the padding off the content on the sides to 0 so the cards take the full width.

File: panels/dev-info/ha-panel-dev-info.html (hunk 1 above)
Comment (postlund) [reply panels/dev-info/ha-panel-dev-info.html]: Changed, I hope I understood you correctly.

File: panels/dev-info/ha-panel-dev-info.html (hunk 2 above)
Comment (postlund) [reply panels/dev-info/ha-panel-dev-info.html]: Same here.

**Comment by postlund** [comment]
I've tried to follow how it's done in `more-info-dialog` but I cannot really seem to get it working the way I want it to. The changes I've made are included in the latest commit. Current behavior for me is that it's horizontally scrollable but not vertically on my phone (I currently have no possibility of testing on desktop). So something is wrong but I don't know what.

**Comment by balloob** [comment]
Tested it locally and seems to work ok.

For future PR: after clear, we should clear the known issues in the browser.

**Comment by balloob** [comment]
Thanks 🌮 🐬

**Comment by postlund** [comment]
🎉🎉🎉

I fixed so that it worked on desktop, but it’s still an issue on my phone. If I tap “Log details”, then I can scroll but it doesn’t work with the content for some reason. A PR that fixes automatic “clear” is in the making!

**Pull request merged**